src/eventDispatchers/mouseEventHandlers/mouseMove.js

```diff
@@ -34,9 +34,9 @@ export default function(evt) {
 
   let imageNeedsUpdate = false;
 
-  // If any tools are active, check if they have a cursor, and if so update image.
+  // If any tools are active, check if they have a canvas cursor, and if so update image.
   if (activeTools.length > 0) {
-    imageNeedsUpdate = activeTools.some(tool => tool.hasCursor);
+    imageNeedsUpdate = activeTools.some(tool => tool.hasCanvasCursor);
```

Comment:
This may not be true if we have svg cursors enabled, and the tool has an svg cursor that takes precedence over its canvas cursor?

Reply:
Perhaps, currently if something has a canvas cursor, it just means we call a refresh on mouse move. A tool could have both, but if the canvas cursor is present, the screen needs to be refreshed?

A canvas cursor can be anything dynamic/animated (i.e. brush/freehandSculpter cursors), or any cursor that needs to be > 128x128 for any reason. In some of these cases you might want an SVG cursor as well.